api/api.go

```diff
@@ -69,6 +69,7 @@ func Setup(ctx context.Context, r *mux.Router, cognitoClient cognito.Client, use
 	r.HandleFunc("/v1/users", contextAndErrors(api.CreateUserHandler)).Methods("POST")
 	r.HandleFunc("/v1/users", contextAndErrors(api.ListUsersHandler)).Methods("GET")
 	r.HandleFunc("/v1/users/{id}", contextAndErrors(api.GetUserHandler)).Methods("GET")
+	r.HandleFunc("/v1/users/{id}", contextAndErrors(api.UpdateUserHandler)).Methods("PUT")
```

Comment:
I'm thinking Matt, it'd probably be good to get some consistency by replacing the inlining of the http verb with `net/http` constants as is the case in other areas of the codebase:

`r.HandleFunc("/v1/users/{id}", contextAndErrors(api.UpdateUserHandler)).Methods(http.MethodPut)`